cmd/promtool/main.go

```diff
@@ -66,15 +65,23 @@ func main() {
 
 	queryCmd := app.Command("query", "Run query against a Prometheus server.")
 	queryInstantCmd := queryCmd.Command("instant", "Run instant query.")
-	queryServer := queryInstantCmd.Arg("server", "Prometheus server to query.").Required().URL()
+	queryServer := queryInstantCmd.Arg("server", "Prometheus server to query.").Required().String()
 	queryExpr := queryInstantCmd.Arg("expr", "PromQL query expression.").Required().String()
 
 	queryRangeCmd := queryCmd.Command("range", "Run range query.")
-	queryRangeServer := queryRangeCmd.Arg("server", "Prometheus server to query.").Required().URL()
+	queryRangeServer := queryRangeCmd.Arg("server", "Prometheus server to query.").Required().String()
 	queryRangeExpr := queryRangeCmd.Arg("expr", "PromQL query expression.").Required().String()
 	queryRangeBegin := queryRangeCmd.Flag("start", "Query range start time (RFC3339 or Unix timestamp).").String()
 	queryRangeEnd := queryRangeCmd.Flag("end", "Query range end time (RFC3339 or Unix timestamp).").String()
 
+	debugCmd := app.Command("debug", "debug cmd")
+	debugPprofCmd := debugCmd.Command("pprof", "pprof cmd")
+	debugPprofServer := debugPprofCmd.Arg("server", "Prometheus server to get pprof files.").Required().String()
+	debugMetricsCmd := debugCmd.Command("metrics", "metrics cmd")
+	debugMetricsServer := debugMetricsCmd.Arg("server", "Prometheus server to get metrics.").Required().String()
+	debugAllCmd := debugCmd.Command("all", "all cmd")
+	debugAllServer := debugAllCmd.Arg("server", "Prometheus server to get all debug infomation.").Required().String()
```

Comment:
"infomation" -> "information"

"to get all debug information" -> "to get all debug information from"